db/seeds.rb

```diff
@@ -9,7 +9,7 @@
 
 pdx_org = Organization.find_or_create_by!(short_name: "pdx_bank") do |organization|
   organization.name = "PDX Diaper Bank"
-  organization.address = "P.O. Box 22613, Portland OR 97269"
+  # organization.address = "P.O. Box 22613, Portland OR 97269"
```

Comment:
I think we can fully remove it. The last migration:

```
class AddAddressFieldsToOrganization < ActiveRecord::Migration[5.1]
  def change
    add_column :organizations, :street, :string
    add_column :organizations, :city, :string
    add_column :organizations, :state, :string
    add_column :organizations, :zipcode, :string
    remove_column :organizations, :address
  end
end
```
removed the address column completely